functions/helloworld/test/sample.system.storage.test.js

```diff
@@ -27,6 +27,12 @@ const gcsFileName = `test-${uuid.v4()}.txt`;
 
 const localFileName = 'test.txt';
 const bucketName = process.env.FUNCTIONS_DELETABLE_BUCKET;
+if (!bucketName) {
+  throw new Error('"FUNCTION_DELETABLE_BUCKET" env var must be set.');
+}
+if (!process.env.GCF_REGION) {
```

Comment:
Nit: should this be its own variable, since it's used in multiple places?

_(If so, apply this comment throughout your PR.)_

Reply:
Leaving for a future update...